addons/custom-fonts/userscript.js

```diff
@@ -0,0 +1,152 @@
+export default async function ({ addon, console }) {
+  await addon.tab.loadScript(`${addon.self.lib}/thirdparty/cs/webfont.js`);
```

Comment:
Where's webfont?

Reply:
You forgot to include this file in the PR, so it won't work

Reply:
Oh, just a sec.

Reply:
I don't have it with me today, I will upload it monday afternoon.